### DISCUSSION TITLE
Can vue 2 components work in vue 3 ?

### DISCUSSION THREAD

**Discussion by LoccBui** [vuetifyjs/vuetify#16845] [Q&A]
Hi everyone, I have a simple question. I want to use the v-data-table of vue 2 replace v-table of vue 3, does it work ?

I've found some other custom component like : v-data-easy-table, v-table-like,... , It's hard to use for me

Does anyone can helpe me ?

Many thanks !

**Comment by abea** [comment]
As far as I'm aware, Vuetify 2 components won't work in Vue 3. They use different Vue APIs.
